SimPEG/potential_fields/base.py

```diff
@@ -14,38 +14,46 @@
 
 ###############################################################################
 #                                                                             #
-#                             Base Potential Fields Simulation                   #
+#                             Base Potential Fields Simulation                #
 #                                                                             #
 ###############################################################################
 
 
 class BasePFSimulation(LinearSimulation):
-    actInd = properties.Array(
-        "Array of active cells (ground)", dtype=(bool, int), default=None
-    )
 
-    n_cpu = properties.Integer(
-        "Number of processors used for the forward simulation",
-        default=int(multiprocessing.cpu_count()),
-    )
+    def __init__(self, mesh, ind_active=None, store_sensitivities='ram', **kwargs):
```

Comment:
I would avoid adding explicit arguments here, and just let the setters handle do the checking/replace when processing the kwargs